run.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+import sys
+from os.path import join, dirname, realpath
+
+base_path = dirname(realpath(__file__))
+source_path = join(base_path, "src", "main", "python")
+script_path = join(base_path, "src", "main", "scripts", "cbas")
+
+sys.path.insert(0, source_path)
+exec(open(script_path, 'r').read())
```

Comment:
I don' like this, what you may want is (this is for zsh):

```
function pyb-env-activate(){
    check_pyb_build || return 1
    local pyb_main_path
    local pyb_script_path
    pyb_main_path=$( pyb -Q print_module_path )
    pyb_script_path=$( pyb -Q print_scripts_path )
    echo "Adding '$pyb_main_path' to PYTHONPATH"
    export PYTHONPATH=$pyb_main_path:$PYTHONPATH
    echo "Adding '$pyb_script_path' to PATH"
    export PATH=$pyb_script_path:$PATH
    rehash
}

function pyb-env-deactivate(){
    # TODO: not sure if this should check if pyb is available
    check_pyb_build || return 1
    local pyb_main_path
    local pyb_script_path
    pyb_main_path=$( pyb -Q print_module_path )
    pyb_script_path=$( pyb -Q print_scripts_path )
    echo "Unsetting PYTHONPATH"
    unset $PYTHONPATH
    echo "Removing $pyb_script_path from PATH"
    path[$path[(i)$pyb_script_path]]=()
    rehash
}

```

That way you can decouple the environment setup from the tool itself. Also you are hardcoding the src and script path, so that will fail if it ever changes. Additionally, when running it like this, you will use the code in `src` and not in `target` thereby loosing any replacements.. e.g. for the version.